packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -195,10 +195,14 @@ export async function generateEmail({user, to, from, subject, bodyComponent, boi
     wordwrap: plainTextWordWrap
   });
   
-  const fromAddress = from || defaultEmailSetting.get()
+  const defaultFromEmail = defaultEmailSetting.get()
+  let fromAddress = from || defaultFromEmail
   if (!fromAddress) {
     throw new Error("No source email address configured. Make sure \"defaultEmail\" is set in your settings.json.");
   }
+  if (isEAForum && fromAddress === defaultFromEmail) {
+    fromAddress = `The EA Forum <${defaultFromEmail}>`
+  }
```

Comment:
To make this uncontroversial I just limited it to EAF, but possibly we'd want this to be the default and use `siteNameWithArticle`?

Reply:
LW already has LessWrong as the from address, do you know how they achieved that?
![Screenshot 2024-06-18 at 11 23 10](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/da6278e8-5ed0-4833-8870-d6a1a16475cb)
![Screenshot 2024-06-18 at 11 23 48](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/55fe4cee-d3d8-4ca4-81ea-6e2d7727553c)

Reply:
Maybe their `defaultEmail` value has the site name in it? @b0b3rt could I ask what you guys are doing here, maybe I missed something?

Reply:
@s-cheng yeah, our `defaultEmail` is `LessWrong <no-reply@lesserwrong.com>`.  `nodemailer`'s docstring for the `from` field: `The e-mail address of the sender. All e-mail addresses can be plain 'sender@server.com' or formatted 'Sender Name sender@server.com'`.  I don't _think_ there's any configuration on the mailgun side to support/enable this, but not 100% sure.

Reply:
Thanks! That makes sense (I thought maybe we were using `defaultEmail` on its own but looks like it's not used anywhere else), so I'll revert this change and just update our settings